eng/Build-Native.cmd

```diff
@@ -29,16 +29,17 @@ if defined VS160COMNTOOLS (
 
 :: Set the default arguments for build
 
-set __BuildArch=x64
-if /i "%PROCESSOR_ARCHITECTURE%" == "amd64" set __BuildArch=x64
-if /i "%PROCESSOR_ARCHITECTURE%" == "x86" set __BuildArch=x86
+set __TargetArch=x64
+if /i "%PROCESSOR_ARCHITECTURE%" == "amd64" set __TargetArch=x64
+if /i "%PROCESSOR_ARCHITECTURE%" == "arm64" set __TargetArch=arm64
+if /i "%PROCESSOR_ARCHITECTURE%" == "x86" set __TargetArch=x86
+set __HostArch=
 set __BuildType=Debug
-set __BuildOS=Windows_NT
-set __Build=1
+set __TargetOS=Windows_NT
```

Comment:
It can be just `windows`, runtime switched targetOS to lower-case and Windows_NT to windows (or win; depending on the context). Diagnostics later only switched unix to lowercase.

Reply:
this likely should be a separate change - although I agree with it.

Reply:
I'm all for putting anymore changes if possible.

Reply:
Yup, it can be a separate change. 👍